Assets/PatchKit Patcher/Editor/CustomBuilding.cs

```diff
@@ -1,58 +1,97 @@
 using UnityEditor;
 using UnityEditor.SceneManagement;
 using System.Linq;
-using System.Diagnostics;
+using UnityEngine.SceneManagement;
 
 namespace PatchKit.Unity
 {
-    public class CustomBuildScripts 
+    public class CustomBuildScripts
     {
-        [MenuItem("Tools/Build/Windows x86")]
-        public static void BuildWindows86 ()
+        [MenuItem("Tools/Build/With build settings/Windows x86")]
+        public static void BuildWindows86()
         {
-            Build(BuildTarget.StandaloneWindows);
+            BuildFromBuildSettings(BuildTarget.StandaloneWindows);
         }
 
-        [MenuItem("Tools/Build/Windows x64")]
-        public static void BuildWindows64 ()
+        [MenuItem("Tools/Build/With build settings/Windows x64")]
+        public static void BuildWindows64()
         {
-            Build(BuildTarget.StandaloneWindows64);
+            BuildFromBuildSettings(BuildTarget.StandaloneWindows64);
         }
 
-        [MenuItem("Tools/Build/Linux x86")]
-        public static void BuildLinux86 ()
+        [MenuItem("Tools/Build/With build settings/Linux x86")]
+        public static void BuildLinux86()
         {
-            Build(BuildTarget.StandaloneLinux);
+            BuildFromBuildSettings(BuildTarget.StandaloneLinux);
         }
 
-        [MenuItem("Tools/Build/Linux x64")]
-        public static void BuildLinux64 ()
+        [MenuItem("Tools/Build/With build settings/Linux x64")]
+        public static void BuildLinux64()
         {
-            Build(BuildTarget.StandaloneLinux64);
+            BuildFromBuildSettings(BuildTarget.StandaloneLinux64);
         }
 
-        [MenuItem("Tools/Build/Linux Universal")]
-        public static void BuildLinux ()
+        [MenuItem("Tools/Build/With build settings/Linux Universal")]
+        public static void BuildLinux()
         {
-            Build(BuildTarget.StandaloneLinuxUniversal);
+            BuildFromBuildSettings(BuildTarget.StandaloneLinuxUniversal);
         }
 
-        [MenuItem("Tools/Build/OSX")]
-        public static void BuildOsx ()
+        [MenuItem("Tools/Build/With build settings/OSX")]
+        public static void BuildOsx()
         {
-            Build(BuildTarget.StandaloneOSXIntel);
+            BuildFromBuildSettings(BuildTarget.StandaloneOSXIntel);
         }
 
-        [MenuItem("Tools/Build/OSX x64")]
-        public static void BuildOsx64 ()
+        [MenuItem("Tools/Build/With build settings/OSX x64")]
+        public static void BuildOsxx64()
         {
-            Build(BuildTarget.StandaloneOSXIntel64);
+            BuildFromBuildSettings(BuildTarget.StandaloneOSXIntel64);
```

Comment:
This option should be the one and only for OSX. Please remove StandaloneOSXIntel and rename "OSX x64" menu item to just "OSX".